main-project/src/users/entity/user.entity.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('users')
+export class Users extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  no: number;
+
+  @Column({ type: 'varchar', length: 50 })
+  email: string;
+
+  @Column({ type: 'tinyint', nullable: true })
+  gender: number;
+
+  @Column({ type: 'varchar', length: 45 })
+  nickname: string;
+
+  @Column({ type: 'tinyint' })
+  admin: number;
```

Comment:
```suggestion
  @Column({ type: 'tinyint' })
  isAdmin: boolean;
```

위와 동일한 이유고 어드민 등급이 여러개 있다면 외래키로 빼서 grade로 관리하는 것도 좋습니다.

Reply:
원래 number타입으로 관리자일 때 1 유저일 때 0을 주려고 했는데 boolean 타입으로 T/F로 하면 될까요?